dfa/utils.py

```diff
@@ -38,3 +38,17 @@ def child_map(word_path):
     paths_ = tree.iddfs(max_depth=max_length, randomize=randomize)
 
     return (word for word, path in paths_ if end is None or path[-1] == end)
+
+def check_equivalence(dfa_a_, dfa_b_):
+    return check_is_subset(dfa_a_, dfa_b_) is None and check_is_subset(dfa_b_, dfa_a_) is None
```

Comment:
Could you make this function have output `None` is they are equiv and the counterexample otherwise?